gax-grpc/src/test/java/com/google/api/gax/grpc/ChannelPoolTest.java

```diff
@@ -395,4 +388,190 @@ public void channelRefreshShouldSwapChannels() throws IOException {
     Mockito.verify(underlyingChannel2, Mockito.only())
         .newCall(Mockito.<MethodDescriptor<String, Integer>>any(), Mockito.any(CallOptions.class));
   }
+
+  @Test
+  public void channelCountShouldNotChangeWhenOutstandingRpcsAreWithinLimits() throws Exception {
+    ScheduledExecutorService executor = Mockito.mock(ScheduledExecutorService.class);
+
+    List<ManagedChannel> channels = new ArrayList<>();
+    List<ClientCall<Object, Object>> startedCalls = new ArrayList<>();
+
+    ChannelFactory channelFactory =
+        () -> {
+          ManagedChannel channel = Mockito.mock(ManagedChannel.class);
+          Mockito.when(channel.newCall(Mockito.any(), Mockito.any()))
+              .thenAnswer(
+                  (Answer<ClientCall<?, ?>>)
```

Comment:
Nit: this line is unnecessary.

Reply:
done

Reply:
Nit: I think there are still two more instances of this in this test class.

Reply:
done